src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +15,33 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    static final String SEARCH_LETTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        ArrayList<String> list = new ArrayList<String>();
+        ArrayList<String> resultList = new ArrayList<String>();
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fileName));
+            while (reader.ready()) {
+                list.add(reader.readLine());
+            }
+
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        } catch (IOException e) {
+            throw new RuntimeException();
+        }
+        for (String str : list) {
+            str = str.toLowerCase();
```

Comment:
let's name this variable so that it will be more clear

Reply:
done